tests/opentracer/test_tracer_gevent.py

```diff
@@ -0,0 +1,192 @@
+import pytest
+import gevent
+import opentracing
+
+from opentracing.ext.scope_manager.gevent import GeventScopeManager
+from tests.opentracer.test_tracer import get_dummy_ot_tracer
+
+
+def get_dummy_gevent_tracer():
+    return get_dummy_ot_tracer('gevent', {}, GeventScopeManager())
+
+
+@pytest.fixture()
+def nop_tracer():
+    return get_dummy_gevent_tracer()
+
+
+class TestTracerGevent(object):
```

Comment:
Not 100% sure what was intended here but you either need to rename or remove this class. Since it has the same name as the class defined on line 49 it just gets overwritten right now.